### PR TITLE
collectors: apps.plugin: apps_groups: update net, aws, ha groups

### PR DESCRIPTION
**Pull request by k0ste**
* added [keepalived](https://github.com/acassen/keepalived) to `ha` group
* added [s3cmd](https://github.com/s3tools/s3cmd) & [s5cmd](https://github.com/peak/s5cmd) tools to `aws-s3` group
* created `netmanager` group, cause `wifi` group was bemused me once - what wifi on my server? Ah, this is [NetworkManager](https://gitlab.freedesktop.org/NetworkManager/NetworkManager) 🤔
* also added [netplan](https://github.com/canonical/netplan) & [systemd-networkd](https://www.freedesktop.org/software/systemd/man/systemd-networkd.service.html#) to `netmanagers`

### CODE REVIEW

File: collectors/apps.plugin/apps_groups.conf [hunk 1]
```diff
@@ -182,7 +183,7 @@ heapster: heapster
 # -----------------------------------------------------------------------------
 # AWS
 
-aws-s3: '*aws s3*'
+aws-s3: '*aws s3*' s3cmd s5cmd
```
Comment (ilyam8) [inline collectors/apps.plugin/apps_groups.conf:186]: should we change the group name? It is `aws-s3`, but we have s**5**cmd?

Comment (k0ste) [reply collectors/apps.plugin/apps_groups.conf:186]: Nope, Amazon S3 is API, s**3**cmd is a old_slow_py client, s**5**cmd is a modern_fast_golang client (see [benchmarks](https://github.com/peak/s5cmd#benchmarks)). Why **5**? Becasue also we have a [s**4**cmd](https://github.com/bloomreach/s4cmd)!

**Review by ilyam8 — APPROVED** [review]

File: collectors/apps.plugin/apps_groups.conf (hunk 1 above)
Comment (ilyam8) [reply collectors/apps.plugin/apps_groups.conf:186]: Thanks! My experience with AWS is only buying items on Amazon UK 😄

**Comment by ilyam8** [comment]
Thanks, @k0ste 👍

**Pull request merged**